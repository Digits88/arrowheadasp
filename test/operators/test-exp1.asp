Test Exponent operator:
<hr>

2 ^ 3 = <%= 2 ^ 3 %><br>
"2" ^ 3 = <%= "2" ^ 3 %><br>
2 ^ "3" = <%= 2 ^ "3" %><br>
"2" ^ "3" = <%= "2" ^ "3" %><br>
2 ^ 0.5 = <%= 2 ^ 0.5 %><br>
IsNull(Null ^ 3) = <%= IsNull(Null ^ 3) %><br>
IsNull(2 ^ Null) = <%= IsNull(2 ^ Null) %><br>
IsNull(Null ^ Null) = <%= IsNull(Null ^ Null) %><br>


