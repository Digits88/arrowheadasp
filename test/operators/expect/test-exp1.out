Test Exponent operator:
<hr>

2 ^ 3 = 8<br>
"2" ^ 3 = 8<br>
2 ^ "3" = 8<br>
"2" ^ "3" = 8<br>
2 ^ 0.5 = 1.414213562373^[0-9]*$<br>
IsNull(Null ^ 3) = True<br>
IsNull(2 ^ Null) = True<br>
IsNull(Null ^ Null) = True<br>


